admin/base/settings/defaults.py

```diff
@@ -41,6 +45,9 @@
     'webpack_loader',
 )
 
+# Custom user model (extends AbstractBaseUser)
+AUTH_USER_MODEL = 'common_auth.MyUser'
```

Comment:
Is this correct? Or does this need to be `common_auth.models.MyUser`? Or does Django do some magic to make the `models` unnecessary?